provides.py

```diff
@@ -38,14 +38,23 @@ def joined_or_changed(self):
         if self._has_auth_request():
             conv.set_state('{relation_name}.auth.requested')
 
-    @hook('{provides:kube-control}-relation-{broken,departed}')
+    @hook('{provides:kube-control}-relation-departed')
     def departed(self):
         """Remove all states.
 
         """
         conv = self.conversation()
         conv.remove_state('{relation_name}.connected')
         conv.remove_state('{relation_name}.gpu.available')
+        conv.set_state('{relation_name}.departed')
```

Comment:
We can probably use the absence of the .connected state. I'll refactor and give it a go.